docs/decisions/0003-reduce-bokchoy-testing.rst

```diff
@@ -0,0 +1,68 @@
+Status
+======
+Draft
+
+
+Context
+=======
+
+edx-platform bokchoy tests are slow, flaky and difficult to debug.  A quick assessment of their value shows that they might be more trouble than they are worth.  And that we might get the same benefit with far fewer tests.
+
+Baseline Data:
+--------------
+
+This data was collected based on the results of bokchoy tests run across all edx-platform PRs over the last 7 days.
+
+* Total number of builds: 253(across 106 PRs)
+* Failures: 49(across 24 PRs)
+
+  * True Failures: 10(across 6 PRs)
+  * Failures that wouldn’t be caught by other test: 3(on 1 PR)
+
+Color
+~~~~~
+
+Of the real failures found, there was one PR which had a failure that was only found via bokchoy and a11y tests.
+    * This PR made a JS change which would have broken many pages from loading.
+
+Recommendation
+==============
+
+Based on the info we have so far, we should only run a suite of smoke tests in bokchoy that ensure the frontend is not entirely broken.
```

Comment:
we should -> we will
_(in order to keep with ADR's active voice and recommended language)_

Plus, there's now a [social direction of no longer using the word `should`](https://www.google.com/search?q=using+the+word+should&oq=using+the+word+should) since it sounds judgemental.